src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -99,6 +99,13 @@
 			<method signature="System.Void .ctor()" />
 		</type>
 
+		<!-- ep-rt-mono.c: ep_rt_mono_init_finish -->
```

Comment:
Shouldn't this be conditional on eventpipe being enabled ?

Reply:
I guess it will be hard to know if we link in things in runtime or not (as components), but there is a "IsSupported" for perf tracing in managed code. How can that be used in the ILLink descriptors?

Reply:
Found it,
feature="System.Diagnostics.Tracing.EventSource.IsSupported"

Reply:
so doing something similar to this then:

<!-- The private Event methods are accessed by private reflection in the base EventSource class. -->
```
  <assembly fullname="System.Private.CoreLib" feature="System.Diagnostics.Tracing.EventSource.IsSupported" featurevalue="true" featuredefault="true">
    <type fullname="System.Diagnostics.Tracing.NativeRuntimeEventSource" />
  </assembly>
```
?

Reply:
but for RuntimeEventSource

Reply:
Yes

Reply:
done